src/ansys/dpf/composites/data_sources.py

```diff
@@ -169,16 +171,24 @@ def _is_matml_file(path: pathlib.Path) -> bool:
     return path.name == _MATML_FILENAME and path.is_file()
 
 
+def _has_1_ext_suffix(path: pathlib.Path) -> bool:
```

Comment:
Nitpick: Purely from the function name, I would expect this to implement `return _1_EXT_SUFFIX in path.suffixes`. But AFAICT, the current implementation is the _correct_ one. Maybe it would be worth adding a short docstring to explain the intended behavior?

Reply:
The suffix is only `_ext`. The .1 is part of the solid model name. For instance, ACPSolidModel_SolidModel.1_ext.h5
 - `ACPSolidModel_` is the prefix for ACP solid models
 - `SolidModel.1` is the name of the solid model
 - `_ext` is there for what ever reason
 - `.h5` is the file extension. 
Here is another example how the file name can look like: `ACPSolidModel_ImportedSolidModel 1_ext.h5`